devops-steps/src/main/java/io/fabric8/kubernetes/pipeline/devops/elasticsearch/ElasticsearchClient.java

```diff
@@ -134,13 +134,12 @@ public static boolean updateEvent(String id, String json, String index, TaskList
         }
     }
 
-    public static boolean isUrlReachable(String url){
-        int timeout = Integer.parseInt(Systems.getEnvVarOrSystemProperty("ES_TIMEOUT","1000")); // default to 1 second
+   public static boolean isUrlReachable(String url) {
+        int timeout = Integer.parseInt(Systems.getEnvVarOrSystemProperty("ES_TIMEOUT", "1000")); // default to 1 second
```

Comment:
Old code will be failing horribly if the property is not parseable. Maybe needs to be fixed in a follow-up